src/server/api/controllers/user.controller.js

```diff
@@ -67,13 +67,22 @@ const createUser = async ({ body }) => {
       role: body.role
     });
 
+  const organization = await knex
+    .from("organizations")
+    .select("*")
+    .where({
+      id: 1
+    });
+
   if (role.length > 0) {
     const hashedPassword = await hashPassword(body.password);
 
     return knex("users").insert({
       role_id: role[0].id,
-      organization_id,
-      email,
+
+      organization_id: organization[0].id,
+      email: email,
+
       password: hashedPassword,
       name,
       status: false
```

Comment:
Looking at the rest of your PR it looks like these changes in the `user.controller.js` was probably just you getting started and then you found out how to do it later. Always make sure to read your PR before starting it so you find out if you are committing something you were not intending to do. Happens to everyone :)

Reply:
modified user.controller